context/app/static/js/helpers/trackers.js

```diff
@@ -36,6 +42,7 @@ const tracker = new MatomoTracker({
   //   setSecureCookie: true,
   //   setRequestMethod: 'POST'
   // }
+  configurations: { setCustomDimension: [1 /* user_type */, getUserType()] },
```

Comment:
So the ID for `user_type` is 1 since this is our first custom dimension?

Reply:
yep.